index.js

```diff
@@ -0,0 +1,114 @@
+const express = require('express');
+const logger = require('morgan');
+const cors = require('cors');
+const udb = require('./data/helpers/userDb.js');
+const pdb = require('./data/helpers/postDb.js');
+
+const server = express();
+server.use(express.json());
+server.use(cors({}));
+
+const errorHelper = (status, message, res) => {
+    res.status(status).json({ error: message });
+};
+
+const lettering = (req, res, next) => {
+    req.body.name = req.body.name.toUpperCase();
+    next();
+};
+
+server.get('/users', (req, res) => {
+    console.log(req.body);
+    udb.get()
+	.then(users => {
+	    console.log(`\n ** users ** \n`, users);
+	    res.json(users);
+	})
+	.catch(err => res.status(500).send(err));
+});
+
+// server.get('/users/:id', (req, res) => {
+//     udb.findById(req.params.id)
+// 	.then(users => {
```

Comment:
Also the reason this get by ID request is not working is due to line 31. For this assignment, there is no findByID helper method. You can check the helper methods by going to `data/helpers/userDb`. There, you'll see that finding by id is actually built in to the get request with an if statement. Basically if an ID is specified, return that ID. Otherwise, return everything. So to sum that up, change line 31 to:
```
udb.get(req.params.id)
```
and it works